src/commands/database-settings-get.ts

```diff
@@ -0,0 +1,60 @@
+import { Response } from "request";
+import * as _ from "lodash";
+import * as request from "request";
+
+import * as responseToError from "../responseToError";
+import * as Command from "../command";
+import * as logger from "../logger";
+import * as FirebaseError from "../error";
+import * as requirePermissions from "../requirePermissions";
+import * as utils from "../utils";
+import * as api from "../api";
+import * as requireInstance from "../requireInstance";
+import { DATABASE_SETTINGS, DatabaseSetting } from "../database/settings";
+
+export default new Command("database:settings:get <path>")
+  .description(
+    "read the realtime database setting. The available settings are:\n" +
```

Comment:
how does "read the realtime database setting" map to "path"? Maybe "read the realtime database setting at `path`" should be the description? it could be cleared up some.

Reply:
Done